internals/secrethub/generate.go

```diff
@@ -137,6 +197,29 @@ func (cmd *GenerateSecretCommand) run() error {
 	return nil
 }
 
+// parseMinFlag takes a min flag value of the following format:
+// <charset>:<count>
+// and returns the charset and count specified in the flag or error
+// if the flag has an invalid format.
+func parseMinFlag(flag string) (randchar.Charset, int, error) {
```

Comment:
We discussed the option of parsing this in a struct that implements the `flag.Value` interface.

Reply:
Although this is a nice improvement, it's not blocking for this PR.